src/sort.cpp

```diff
@@ -24,15 +24,15 @@
 namespace smt {
 
 const std::unordered_map<SortKind, std::string> sortkind2str(
-    { { ARRAY, "ARRAY" },
-      { BOOL, "BOOL" },
-      { BV, "BV" },
-      { INT, "INT" },
-      { REAL, "REAL" },
-      { FUNCTION, "FUNCTION" },
-      { UNINTERPRETED, "UNINTERPRETED" },
-      { UNINTERPRETED_CONS, "UNINTERPRETED_CONS" },
-      { DATATYPE, "DATATYPE" } });
+    { { ARRAY, "Array" },
+      { BOOL, "Bool" },
+      { BV, "BitVec" },
+      { INT, "Int" },
+      { REAL, "Real" },
+      { FUNCTION, "Function" },
+      { UNINTERPRETED, "Uninterpreted" },
+      { UNINTERPRETED_CONS, "UninterpretedSortConstructor" },
```

Comment:
Out of curiosity: What is the purpose of those two kinds?

Reply:
Good question, it might not be the best solution. Essentially it distinguishes between an uninterpreted sort, and a parameterized uninterpreted sort before it has been instantiated. So if we had
```
(declare-sort U 3)
```
Then `U` would have `SortKind` `UNINTERPRETED_CONS`, but `(U Bool Bool Bool)` would have `SortKind` `UNINTERPRETED`.